packages/webapp/src/components/Form/SexDetails/index.tsx

```diff
@@ -71,8 +73,12 @@ export default function SexDetails({
         error={total > maxCount ? t('ADD_ANIMAL.SEX_DETAIL_ERROR', { count: maxCount }) : undefined}
         showResetIcon={false}
         rightSection={<ChevronDown className={styles.chevronDownIcon} />}
+        disabled={isDisabled}
         mainSection={
-          <TextButton onClick={(e) => setAnchor(e.currentTarget)} className={styles.button}>
+          <TextButton
+            onClick={isDisabled ? () => {} : (e) => setAnchor(e.currentTarget)}
+            className={styles.button}
```

Comment:
Thank you for looking into the focus style!
I was curious about what was going on and looked into the code.
This TextButton is still being focused while `InputBase` itself is disabled, so adding `disabled={isDisabled}` here should fix the behaviour without the CSS!

I'll approve but won't merge in case you want to make the change :)

Reply:
Thank you so much for investigating that!! Yes I definitely prefer that and will push it now 🙏